webrender/src/frame.rs

```diff
@@ -976,7 +978,9 @@ impl Frame {
                                      display_lists,
                                      device_pixel_ratio,
                                      texture_cache_profile);
-        resource_cache.expire_old_resources(self.id);
+        // Expire any resources that haven't been used for CACHE_EXPIRY_TIME frames.
+        let expiry_frame = FrameId(cmp::max(CACHE_EXPIRY_TIME, self.id.0) - CACHE_EXPIRY_TIME);
```

Comment:
I'd prefer us to not do any expiration before we reach `CACHE_EXPIRY_TIME`, since we may load a lot of stuff at frame 0, and this logic would be happy to get rid of it instantly.
This is still a big improvement though, so we can get it patched later on.